xla/xla_data.proto

```diff
@@ -1039,6 +1039,7 @@ message ParameterReplication {
 message WhileLoopBackendConfig {
   message KnownTripCount {
     int64 n = 1;
+    optional int64 step = 2; // Value of k if the update operation is i+=k
```

Comment:
Could you expand a bit more on the semantics of this field? Maybe take it out into a separate PR?

Reply:
Removed those changes and computing step within the pass.